src/main/java/com/google/maps/model/PlaceDetails.java

```diff
@@ -0,0 +1,197 @@
+     * Google-scoped place ID after passing the moderation process. */
+    public String placeId;
+
+    /** scope — The scope of an alternative place ID will always be APP, indicating that the
+     * alternative place ID is recognised by your application only. */
+    public PlaceIdScope scope;
+  }
+
+  /** altIds is an optional array of alternative place IDs for the place, with a scope related to
+   * each alternative ID. */
+  public AlternatePlaceIds[] altIds;
+
+  public enum PriceLevel {
+    FREE, INEXPENSIVE, MODERATE, EXPENSIVE, VERY_EXPENSIVE,
+
+    /**
+     * Indicates an unknown price level type returned by the server. The Java Client for Google Maps
+     * Services should be updated to support the new value.
+     */
+    UNKNOWN
+  }
+
+  /** priceLevel is the price level of the place. The exact amount indicated by a specific value
+   * will vary from region to region. */
+  public PriceLevel priceLevel;
+
+  /** rating contains the place's rating, from 1.0 to 5.0, based on aggregated user reviews. */
+  public float rating;
+
+  /** userRatingsTotal is an undocumented field.
+   *
+   * @deprecated this field is not documented.
+   */
+  public int userRatingsTotal;
+
+
+  static public class Review {
+    static public class AspectRating {
+      public enum RatingType {
+        APPEAL, ATMOSPHERE, DECOR, FACILITIES, FOOD, OVERALL, QUALITY, SERVICE,
+
+        /**
+         * Indicates an unknown rating type returned by the server. The Java Client for Google Maps
+         * Services should be updated to support the new value.
+         */
+        UNKNOWN
+      }
+
+      /** type is the name of the aspect that is being rated. */
+      public RatingType type;
+
+      /** rating is the user's rating for this particular aspect, from 0 to 3. */
+      public int rating;
+    }
+
+    /** aspects contains a collection of AspectRating objects, each of which provides a rating of
+     * a single attribute of the establishment. */
+    public AspectRating[] aspects;
+
+    /** authorName the name of the user who submitted the review. Anonymous reviews are attributed
+     * to "A Google user". */
+    public String authorName;
+
+    /** authorUrl the URL to the users Google+ profile, if available. */
+    public URL authorUrl;
+
+    /** language an IETF language code indicating the language used in the user's review. */
+    public String language;
+
+    /** rating the user's overall rating for this place. This is a whole number, ranging from 1 to
+     * 5. */
+    public int rating;
+
+    /** text is the user's review. When reviewing a location with Google Places, text reviews are
+     * considered optional. */
+    public String text;
+
+    /** time is the time that the review was submitted, as seconds since epoch.*/
+    public long time; // TODO(brettmorgan): figure out how to convert this to a DateTime.
+  }
+
+  /** reviews is an array of up to five reviews. */
```

Comment:
Will this always be five? Could you just say "It will contain a small number of..." (happy to defer to other libs if they do something different)